wgpu-core/src/hub.rs

```diff
@@ -89,55 +89,67 @@ impl IdentityManager {
 #[derive(Debug)]
 pub struct Storage<T, I: TypedId> {
     //TODO: consider concurrent hashmap?
-    map: VecMap<(T, Epoch)>,
+    map: VecMap<(T, Epoch, Backend)>,
```

Comment:
this is unexpected, why do we need to add the backend here?
Each storage, conceptually, only stores things for the same backend. There can't be two different indices of that map with different backends.

Reply:
Okay, I thought as much, but good to know! I will have to figure out a way to produce the `TypedId` for the iterator (need a backend to `zip` the `TypedId`), but maybe I can just pass the backend to `iter()`...